Bottom: 1e3526d39b2591bb10a08bfed2622f05f82b1bd8
Top:    fb3f150e6146757e6bdb884182c4b19457bd6b6b
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-11 10:41:02 -0700

rstream: Clarify use of async test option

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/examples/rstream.c b/examples/rstream.c
index 7f39b29..decb204 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -561,7 +561,7 @@ int main(int argc, char **argv)
 			printf("\t[-p port_number]\n");
 			printf("\t[-T test_option]\n");
 			printf("\t    s|sockets - use standard tcp/ip sockets\n");
-			printf("\t    a|async - asynchronous operation\n");
+			printf("\t    a|async - asynchronous operation (use poll)\n");
 			printf("\t    b|blocking - use blocking calls\n");
 			printf("\t    n|nonblocking - use nonblocking calls\n");
 			printf("\t    v|verify - verify data\n");
